snap/snapcraft.yaml

```diff
@@ -17,7 +17,8 @@ apps:
       - network-bind
     environment:
       PATH: $SNAP/bin/:$PATH
-      PYTHONPATH: $SNAP:$SNAP/lib/python3.8/site-packages
+      PYTHONPATH: $SNAP:$SNAP/lib/python3.6/site-packages
+    extensions: [flutter-master]
```

Comment:
What do we exactly need this one for? Is it just providing the baseline tools in order to do a `flutter build`?

Reply:
Yes, exactly. 
Please check the following link for more details
https://snapcraft.io/docs/flutter-extension